docs/os/build-details.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Build details
+parent: Operating System
+nav_order: 210
```

Comment:
nav_order should be between 310 (Building the Armbian base image) and 320 (Configuration), so maybe 315?

Reply:
Done.